modules/OCPPConfiguration/main/mapping_reader.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "mapping_reader.hpp"
+#include "util.hpp"
+#include <iostream>
```

Comment:
put includes into sections

Reply:
Done but I think this should be done automatically by some tool instead of having to do it manually. Isn't this set up somehow already?